engine/deck.go

```diff
@@ -1,17 +1,35 @@
 package engine
 
-import "errors"
+import (
+	"errors"
+	"math/rand"
+	"time"
+)
 
-// Deal takes a pointer to deck (slice) of Cards and returns a slice of numDealt 
-// Cards from it. If the slice of cards is not big enough, it returns an empty 
+// Deal takes a pointer to deck (slice) of Cards and returns a slice of numDealt
+// Cards from it. If the slice of cards is not big enough, it returns an empty
 // slice and an error. Deck is reduced in size by numDealt.
-
-func Deal (numDealt uint, deck *[]Card) (hand []Card, err error) {
+func Deal(numDealt uint, deck *[]Card) (hand []Card, err error) {
 	d := *deck
 	if len(d) < int(numDealt) {
 		return hand, errors.New("Deal: not enough cards in deck")
 	}
 	hand = d[:numDealt]
 	*deck = d[numDealt:]
 	return
-}
\ No newline at end of file
+}
+
+func shuffle(cards []Card) {
```

Comment:
Is this going to be exported? If so, we should cap it. If not, then deal shouldn't be exported or cap'd either.